Refresh images in Scout dashboard (#17638)
docker/docs: scout/dashboard.md

## Repository settings

> ![A screenshot showing where to find the settings page for Docker Scout](./images/dashboard-overview-settings.png)

Reviewer: Do we really need this screenshot here? I believe our users are smart enough to locate the settings icon by reading the description below.

Reviewer: Docker Desktop has the same icon. It's called the "Settings" icon- https://docs.docker.com/desktop/settings/mac/ , which is generic enough for non-native speakers. Let's use the settings icon to be consistent.

- The versions of the package used by images in the organization.
- The number of images that use the package.

## Base images

> The **Base images** tab shows all base images images are based upon in an organization.

Reviewer 2: **nitpick:** "base images images are based upon" sounds like a mouthful 😬 

**suggestion:** 
> The **Base images** tab shows all base images used in an organization.

In my opinion, I could be wrong, the term `base image` is pretty much self-explanatory so we shouldn't have to extend the sentence

Author: @fdiogo It does sound like a mouthful, but I am not sure it is completely clear as a concept… I guess it is really.

```suggestion
The **Base images** tab shows all base images images used by repositories in an organization.
```

How about that?

Reviewer 2: We still have the double `images` but it is better 👍

Author: Yeah, I know :( Let's use it for now.
